src/pykeen/sampling/negative_sampler.py

```diff
@@ -25,14 +25,23 @@ def __init__(
         self,
         triples_factory: TriplesFactory,
         num_negs_per_pos: Optional[int] = None,
+        filtered: bool = None,
+        corruption_scheme: Tuple[str] = None,
     ) -> None:
         """Initialize the negative sampler with the given entities.
 
         :param triples_factory: The factory holding the triples to sample from
         :param num_negs_per_pos: Number of negative samples to make per positive triple. Defaults to 1.
+        :param filtered: Whether proposed corrupted triples that are in the training data should be filtered.
+            Defaults to False.
+        :param corruption_scheme: What sides (h, r, t) should be corrupted. Defaults to head and tail (h, t).
```

Comment:
The type annotation here is wrong - `Tuple[str]` means a 1-tuple with a str. A `Tuple[str, ...]` means a variable length tuple of strs, but then it's better to use a `Sequence[str]` which means ordered finite length (list and tuple both count) or even `Set[str]` since in this case order is unimportant and uniqueness is. Please update the docstring also to have examples that are valid python - the strings should be quoted

Reply:
Collection is also possible for unordered collection :wink: